manifests/a/AnyDeskSoftwareGmbH/AnyDesk/9.0.2/AnyDeskSoftwareGmbH.AnyDesk.installer.yaml

```
@@ -24,14 +25,13 @@ Installers:
   ElevationRequirement: elevationRequired
 - InstallerLocale: en-US
   Architecture: x86
-  InstallerType: msi
+  InstallerType: wix
   InstallerUrl: https://download.anydesk.com/AnyDesk.msi
   InstallerSha256: 9F32CDA20CAC7AFAB4BF2ECFD2B080CC4F1F700DFF13566468DA69059B7EBF03

```

```suggestion
 InstallerSha256: 3ADEC63D12273021DFB5B8998B934E659B12CA1C2F799C7899DC7FEFF9EC7A0C
```

(Automated response - build 953.)